Library/Homebrew/test/support/fixtures/internal_tap_json/homebrew-core.json

```diff
@@ -110,7 +91,29 @@
             "sha256": "ab49318d75eed3ee932c8e5add22f252ec0c852aad94945022877f926e93899f"
           }
         }
+      },
+      "dependencies": {
+        "cmake": {
+          "tags": [
+            "build"
+          ]
+        },
+        "python@3.12": {
+          "tags": [
+            "build"
+          ]
+        },
+        "llvm": {
+          "tags": [
+            "build",
+            "test"
+          ],
+          "uses_from_macos": null
+        },
+        "zlib": {
+          "uses_from_macos": null
+        }
```

Comment:
It might make sense to add another formula that has uses from macOS bounds to make sure that's working correctly. Or I guess I could replace one of these examples too in a pinch.

Reply:
@apainintheneck Yeh, good idea 👍🏻